src/config.xml

```diff
@@ -1809,6 +1809,22 @@ to disable this feature.
  be put under the same header in the alphabetical index.
  The \c IGNORE_PREFIX tag can be used to specify a prefix
  (or a list of prefixes) that should be ignored while generating the index headers.
+]]>
+      </docs>
+    </option>
+    <option type='int' id='MAX_ITEMS_BEFORE_QUICK_INDEX' minval='0' maxval='3000' defval='30'>
+      <docs>
+<![CDATA[
+ MAX_ITEMS_BEFORE_QUICK_INDEX defines, when to set the global lists of functions, 
+ variables, typdefs, enums, macros grouped into sections by starting letter. Default value: 30
+]]>
```

Comment:
Default value should not be mentioned (is automatically added when building the documentation etc.)

The formulation of the sentence is not correct, to be honest I don't understand it. Furthermore the `MAX_ITEMS_BEFORE_QUICK_INDEX` should be preceded by `The \c `. See formulations of other items.

A reference to `MAX_ITEMS_BEFORE_MULTIPAGE_INDEX` should be present.